### ISSUE TITLE
How can i use getRowData

### ISSUE THREAD

**Issue by ppingliu** [jspreadsheet/ce#64]
Hi Paul
i have a other questions;
i need get getData like  "$('#my').jexcel('getData')"  is ok.
but i need get  getRowData   "$('#my').jexcel('getRowData')" is failed.
why?

thank you a again.

**Comment by pphod** [comment]
You should define, which row you would like to load.

$('#my').jexcel('getRowData', 1); // (The number starts from zero, so one means second row);

**Comment by ppingliu** [comment]
thank you very much

how can i change the "1";  .prop('id') or (row)?

**Comment by pphod** [comment]
Sorry, I am afraid did not get your question. Can you rephrase please?

**Comment by ppingliu** [comment]
OK!
i meaning when i click this cell, i hope get this cell the entire row of data

**Comment by ppingliu** [comment]
like this $('#my').jexcel('getRowData', 'row('id')')

**Comment by pphod** [comment]
No, you need to add a number you would like to read.

$('#my').jexcel('getRowData', [integer])

**Comment by lodhaneha7** [comment]
Hi Paul,
       Please let me, how can I use $('#my').jexcel('getRowData', 'row('id')') this method in angular?
